demo/lib/distro.bash

```diff
@@ -494,6 +500,66 @@ EOF
     fi
 }
 
+fedora33-image-url() {
+    echo "https://download.fedoraproject.org/pub/fedora/linux/releases/33/Cloud/x86_64/images/Fedora-Cloud-Base-33-1.2.x86_64.qcow2"
+}
+
+fedora33-ssh-user() {
+    fedora-ssh-user
+}
+
+fedora33-install-utils() {
+    fedora-install-utils
+}
+
+fedora33-install-repo() {
+    fedora-install-repo $*
+}
+
+fedora33-install-pkg() {
+    fedora-install-pkg $*
+}
+
+fedora33-remove-pkg() {
+    fedora-remove-pkg $*
+}
+
+fedora33-install-pkg-local() {
+    fedora-install-pkg-local $*
+}
+
+fedora33-install-golang() {
+    fedora-install-golang $*
+}
+
+fedora33-install-crio-pre() {
+    fedora-install-crio-version 1.20
+}
+
+fedora33-install-crio() {
+    fedora-install-crio $*
+}
+
+fedora33-install-containerd-pre() {
+    fedora-install-containerd-pre
+}
+
+fedora33-install-containerd-post() {
+    fedora-install-containerd-post
+}
+
+fedora33-config-containerd-post() {
+    fedora-config-containerd-post
+}
+
+fedora33-install-k8s() {
+    fedora-install-k8s $*
+}
+
+fedora33-bootstrap-commands-post() {
+    fedora-bootstrap-commands-post $*
+}
```

Comment:
nitpick: I think we should name support for F33 `fedora-33` instead of `fedora33`. Then of these functions we could leave out those which are just tautologic fallbacks to the unversioned fedora ones, because `distro-resolve-fn()` has built-in support for that: it always looks for `distro-function()` next if `distro-function-XY()` is not defined.

Reply:
> distro-resolve-fn() has built-in support for that: it always looks for distro-function() next if distro-X-function() is not defined.

Ah, I missed that. Indeed that change makes sense, I will send a new version.